doc/api/assert.md

```diff
@@ -6,6 +6,44 @@
 
 The `assert` module provides a simple set of assertion tests that can be used to
 test invariants.
+A `strict` and a `legacy` mode exist, while it is recommended to only use
+[`strict mode`][].
+
+## Strict mode
+
+When using the strict mode any assert function will use the equality used in the
+strict function mode. So [`assert.deepEqual`][] will for example work the same
+as [`assert.deepStrictEqual`][].
+
+It can be accessed using:
+
+```js
+const assert = require('assert').strict;
+```
+
+## Legacy mode
+
+> Stability: 0 - Deprecated: Use strict mode instead.
+
+When accessing `assert` directly instead of using the `strict` property the
+"non-strict" equality will be used for the "non-strict" function names e.g.
+[`assert.deepEqual`][].
```

Comment:
Nit: Put everything after `e.g.` and up to the closing `.` in parentheses.

Nit: Here and several places above, function names should be followed by `()` I believe. So `assert.deepEqual()` rather than `assert.deepEqual`. That said, if there's already a lot of no-parentheses examples in the doc, never mind. :-D